addons/website/views/snippets/s_popup.xml

```diff
@@ -42,19 +42,20 @@
                 <we-button data-select-class="s_popup_middle" data-name="position_middle">Middle</we-button>
                 <we-button data-select-class="s_popup_bottom" data-name="position_bottom">Bottom</we-button>
             </we-select>
-            <we-select string="Size" data-apply-to=".modal-dialog" data-name="s_popup_size_opt">
+            <we-select string="Size" data-apply-to=".modal-dialog" data-name="s_popup_size_opt" data-trigger="s_popup_backdrop_color_opt">
```

Comment:
why not data-trigger="popup_backdrop_opt" ?
and data-trigger-value="false" for the s_popup_size_full button
Is it possible ?